project/plugins.sbt

```diff
@@ -1,5 +1,3 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
 
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.3")
```

Comment:
[`scalariform`](https://github.com/scala-ide/scalariform) is no longer maintained, [`scalafmt`](https://scalameta.org/scalafmt/) is the Scala source code formatter of choice these days!

I'm removing scalariform here as I'm fairly ambivalent about source-code formatters, and scalariform does not work with the updated build due to a dependency clash (on `scala-xml`, I think).